runtime/altair/src/lib.rs

```diff
@@ -132,6 +132,8 @@ pub fn native_version() -> NativeVersion {
 	}
 }
 
+const MAX_BLOCK_WEIGHT: Weight = MAXIMUM_BLOCK_WEIGHT.set_proof_size(MAX_POV_SIZE as u64);
+
 parameter_types! {
 	pub const MaximumBlockWeight: Weight = MAXIMUM_BLOCK_WEIGHT;
```

Comment:
This is not needed anymore right?

Reply:
Good catch, I had spaced that.  It looks like it's actually being used once later for the scheduler config.

I've removed that in the scheduler config in the work to have the max proof size pulled from the relay chain here https://github.com/centrifuge/centrifuge-chain/pull/1134.